### ISSUE TITLE
Space between annotation and property declaration does not generate warning

### ISSUE THREAD

**Issue by vielasis** [pinterest/ktlint#559]
I assume this is a bug because the IDE (IntelliJ) auto-formats this properly.

```
// Actual
data class A(
    @Annotation(...)val property: Type 
)
```

Is not flagged by ktlint 0.34.0

```
// Expected proper formatting
data class A(
    @Annotation(...) val property: Type 
)
```
